react-accordion-component/my-app/src/Accordion.css

```diff
@@ -0,0 +1,25 @@
+.row {
+  width: 350px;
+}
+
+.btn-style {
+  display: flex;
+  border: 2px solid black;
+  font-size: x-large;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  background-color: rgb(214, 214, 214);
+  height: 40px;
+  width: 341px;
+  cursor: pointer;
+  justify-content: left;
+  align-items: center;
+  padding-left: 5px;
+}
+
+p {
```

Comment:
Imagine of this was a larger app? It's probably better to target with a `class` instead! This would change all `p` tags!